bootstrap-joining-demo/dns-api-mesos/README.md

```diff
@@ -13,4 +13,14 @@ Push image into DockerHub
 Mesosphere DC/OS
 ================
 
-Use `mesos.host-mode.json` service description.
\ No newline at end of file
+Host mode
+---------
+Use `marathon/app.host-mode.json` service description.
```

Comment:
What do those modes mean, could you add a short explanation? Like 1 sentence or two..?